backend/corpora/common/corpora_config.py

```diff
@@ -4,11 +4,18 @@
 
 
 class CorporaConfig(SecretConfig):
+    environ_source = "CORPORA_CONFIG"
+
     def __init__(self, *args, **kwargs):
         super().__init__("backend", secret_name="config", **kwargs)
 
+    def get_defaults_template(self):
+        return {"upload_file_formats": ["h5ad"], "upload_max_file_size": 30}
```

Comment:
nit: do we need to specify the unit is GB?

e.g., `upload_max_file_size_gb`

Reply:
I'll change it to gb